uptrain/operators/code/detection.py

```diff
@@ -0,0 +1,57 @@
+"""
+Implement operator to detect code in a response.
+"""
+
+from __future__ import annotations
+import typing as t
+
+from loguru import logger
+import polars as pl
+
+if t.TYPE_CHECKING:
+    from uptrain.framework.base import *
+from uptrain.operators.base import *
+
+
+@register_op
+class CodeIdentificationScore(ColumnOp):
+    """
+    
+    Go through the response and identify if there is any code in it.
```

Comment:
Does it also give the identified code segment?

Reply:
Yes, will update it accordingly